airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/error_handlers/http_response_filter.py

```diff
@@ -56,7 +56,7 @@ def matches(self, response: requests.Response) -> Optional[ResponseAction]:
             return None
 
     def _response_matches_predicate(self, response: requests.Response) -> bool:
-        return self.predicate and self.predicate.eval(None, response=response.json())
+        return self.predicate and self.predicate.eval(None, response=response.json(), headers=response.headers)
```

Comment:
should we update the "link" header here to be consistent with the cursor pagination strategy?